frontend/src/pull.ts

```diff
@@ -187,6 +187,10 @@ export class Pull extends PullData {
     const linkIdPrefix = isReview ? "pullrequestreview" : "issuecomment";
     return this.url() + `#${linkIdPrefix}-${sig.data.comment_id}`;
   }
+
+  linkToBranch() {
```

Comment:
Oh, this name is a bit off and implies that this links to the pull requests own branch. Maybe this should be `getUrlOfBaseBranch()` or something similar.

Reply:
Updated via https://github.com/iFixit/pulldasher/pull/362/commits/ee3cf2a3284009e61fe2664a17fee2ffab886b03